content/news/2022-01-06-bevy-0.6/faster_gltf.svg

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Unless GitHub's UI is decieving me (which is very possible) this image isn't used

(To be clear, I think this not being used is probably good, but its inclusion is still not needed)

Reply:
Yeah this was a "straggler" that I forgot to remove. I realized that while the story it told was "gltf loading goes brrr", it wasn't a particularly useful / accurate depiction of the situation.